packages/dart/lib/src/network/dio_adapter_js.dart

```diff
@@ -1,9 +1,7 @@
-import 'dart:io';
-
-import 'package:dio/adapter_browser.dart';
+import 'package:dio/browser.dart';
 import 'package:dio/dio.dart';
 
-HttpClientAdapter createHttpClientAdapter(SecurityContext? securityContext) {
+HttpClientAdapter createHttpClientAdapter(dynamic securityContext) {
```

Comment:
If we will not use the `securityContext` do you think it is better to make it `_`?
like this:
```dart 
HttpClientAdapter createHttpClientAdapter(_) { /* code*/}

```

Reply:
In some parts, we need it variable

Reply:
I suggest the file be like this:
 
```dart
import 'package:dio/adapter_browser.dart';
import 'package:dio/dio.dart';

HttpClientAdapter createHttpClientAdapter(_) {
  return BrowserHttpClientAdapter();
}
 
```
What do you think?

Reply:
Your suggestion is currently being used in this [file](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/packages/dart/lib/src/network/dio_adapter_js.dart)